### ISSUE TITLE
Assertion 'link->route_messages > 0' failed at src/network/networkd-route.c:1140, function int route_handler(sd_netlink *, sd_netlink_message *, Link *)(). Aborting.

### ISSUE THREAD

**Issue by mrc0mmand** [systemd/systemd#17396]
**systemd version the issue has been seen with**
 > latest master

**Used distribution**
 > Arch Linux

**Linux kernel version used** (`uname -a`)
 > 5.7.12

**CPU architecture issue was seen on**
 > x86_64

The `systemd-networkd-tests.py` testsuite occasionally fails due to failed assertion during `test_route_static`:

```
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Remembering route: dst: 2001:1234:5:8f63::1/128, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: local, proto: kernel, type: local
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Remembering updated address: 149.10.124.58/28 (valid forever)
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: link_check_ready(): static addresses are not configured.
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Remembering route: dst: 149.10.124.58/32, src: n/a, gw: n/a, prefsrc: 149.10.124.58, scope: host, table: local, proto: kernel, type: local
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Remembering route: dst: 149.10.124.63/32, src: n/a, gw: n/a, prefsrc: 149.10.124.58, scope: link, table: local, proto: kernel, type: broadcast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Remembering route: dst: 149.10.124.48/28, src: n/a, gw: n/a, prefsrc: 149.10.124.58, scope: link, table: main, proto: kernel, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Remembering route: dst: 149.10.124.48/32, src: n/a, gw: n/a, prefsrc: 149.10.124.58, scope: link, table: local, proto: kernel, type: broadcast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Addresses set
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 202.54.1.3/32, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main, proto: static, type: unreachable
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 149.10.123.3/32, src: n/a, gw: n/a, prefsrc: n/a, scope: link, table: local, proto: static, type: broadcast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 149.10.123.4/32, src: n/a, gw: n/a, prefsrc: n/a, scope: link, table: main, proto: static, type: multicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 202.54.1.2/32, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main, proto: static, type: blackhole
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 149.10.123.2/32, src: n/a, gw: n/a, prefsrc: n/a, scope: link, table: local, proto: static, type: anycast
Oct 19 21:46:54 arch.localdomain systemd-resolved[160261]: Sent message type=signal sender=n/a destination=n/a path=/org/freedesktop/resolve1 interface=org.freedesktop.DBus.Properties member=PropertiesChanged cookie=3848 reply_cookie=0 signature=sa{sv}as error-name=n/a error-message=n/a
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 2001:1234:5:9fff:ff:ff:ff:ff/128, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 149.10.123.1/32, src: n/a, gw: n/a, prefsrc: n/a, scope: host, table: local, proto: static, type: local
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 192.168.1.2/32, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: n/a, src: n/a, gw: n/a, prefsrc: n/a, scope: link, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 202.54.1.4/32, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main, proto: static, type: prohibit
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 192.168.1.1/32, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 169.254.0.0/16, src: n/a, gw: n/a, prefsrc: n/a, scope: link, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 149.10.124.64/32, src: n/a, gw: n/a, prefsrc: n/a, scope: link, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 2001:1234:5:8fff:ff:ff:ff:ff/128, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: n/a, src: n/a, gw: 149.10.125.65, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: n/a, src: n/a, gw: 149.10.124.64, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 192.168.10.1/32, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 2001:1234:5:7fff:ff:ff:ff:ff/128, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: n/a, src: n/a, gw: 2001:1234:5:8fff:ff:ff:ff:ff, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 192.168.10.2/32, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Could not add route: File exists
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Could not set routes: File exists
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Failed
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: State changed: configuring -> failed
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: Sent message type=signal sender=n/a destination=n/a path=/org/freedesktop/network1/link/_3361 interface=org.freedesktop.DBus.Properties member=PropertiesChanged cookie=65 reply_cookie=0 signature=sa{sv}as error-name=n/a error-message=n/a
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: Received remembered route: dst: 202.54.1.3/32, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main, proto: static, type: unreachable
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Received remembered route: dst: 149.10.123.3/32, src: n/a, gw: n/a, prefsrc: n/a, scope: link, table: local, proto: static, type: broadcast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Received remembered route: dst: 149.10.123.4/32, src: n/a, gw: n/a, prefsrc: n/a, scope: link, table: main, proto: static, type: multicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: Received remembered route: dst: 202.54.1.2/32, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main, proto: static, type: blackhole
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Received remembered route: dst: 149.10.123.2/32, src: n/a, gw: n/a, prefsrc: n/a, scope: link, table: local, proto: static, type: anycast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Received remembered route: dst: 2001:1234:5:9fff:ff:ff:ff:ff/128, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Received remembered route: dst: 149.10.123.1/32, src: n/a, gw: n/a, prefsrc: n/a, scope: host, table: local, proto: static, type: local
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Received remembered route: dst: 192.168.1.2/32, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Received remembered route: dst: n/a, src: n/a, gw: n/a, prefsrc: n/a, scope: link, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: Received remembered route: dst: 202.54.1.4/32, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main, proto: static, type: prohibit
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Received remembered route: dst: 192.168.1.1/32, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Received remembered route: dst: 169.254.0.0/16, src: n/a, gw: n/a, prefsrc: n/a, scope: link, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Received remembered route: dst: 149.10.124.64/32, src: n/a, gw: n/a, prefsrc: n/a, scope: link, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Received remembered route: dst: 2001:1234:5:8fff:ff:ff:ff:ff/128, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Received remembered route: dst: n/a, src: n/a, gw: 149.10.125.65, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Received remembered route: dst: n/a, src: n/a, gw: 149.10.124.64, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Received remembered route: dst: 192.168.10.1/32, src: n/a, gw: 149.10.124.59, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Received remembered route: dst: 192.168.10.1/32, src: n/a, gw: 149.10.124.60, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-resolved[160261]: Sent message type=signal sender=n/a destination=n/a path=/org/freedesktop/resolve1 interface=org.freedesktop.DBus.Properties member=PropertiesChanged cookie=3849 reply_cookie=0 signature=sa{sv}as error-name=n/a error-message=n/a
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Received remembered route: dst: 2001:1234:5:7fff:ff:ff:ff:ff/128, src: n/a, gw: 2001:1234:5:8fff:ff:ff:ff:ff, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Received remembered route: dst: 2001:1234:5:7fff:ff:ff:ff:ff/128, src: n/a, gw: 2001:1234:5:9fff:ff:ff:ff:ff, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Received remembered route: dst: n/a, src: n/a, gw: 2001:1234:5:8fff:ff:ff:ff:ff, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Received remembered route: dst: 192.168.10.2/32, src: n/a, gw: 2001:1234:5:8fff:ff:ff:ff:ff, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Received remembered route: dst: 192.168.10.2/32, src: n/a, gw: 2001:1234:5:9fff:ff:ff:ff:ff, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: Assertion 'link->route_messages > 0' failed at src/network/networkd-route.c:1140, function int route_handler(sd_netlink *, sd_netlink_message *, Link *)(). Aborting.
```

Most recently seen in #17395 and #17392.

Related job logs:
 * https://jenkins-systemd.apps.ocp.ci.centos.org/job/upstream-vagrant-archlinux/329/artifact//systemd-centos-ci/index.html
 * https://jenkins-systemd.apps.ocp.ci.centos.org/job/upstream-vagrant-archlinux-sanitizers/337/artifact//systemd-centos-ci/index.html

/cc @yuwata

**Comment by yuwata** [comment]
Hmm, from https://jenkins-systemd.apps.ocp.ci.centos.org/job/upstream-vagrant-archlinux-sanitizers/337/artifact//systemd-centos-ci/index.html,
```
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: New device has no master, continuing without
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: Flags change: +BROADCAST +NOARP
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: Link 362 added
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: udev initialized link
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: State changed: pending -> initialized
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: netdev has index 362
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: netdev ready
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: Saved original MTU: 1500
(snip)
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: New device has no master, continuing without
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: Flags change: +BROADCAST +NOARP
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: Link 361 added
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: Could not find device, waiting for device initialization: No such device
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: Could not set ifindex to 361, already set to 362
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: Could not process new link message for netdev, ignoring: File exists
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: Forgetting route: dst: 2001:1234:5:7fff:ff:ff:ff:ff/128, src: n/a, gw: 2001:1234:5:8fff:ff:ff:ff:ff, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: Forgetting route: dst: 2001:1234:5:7fff:ff:ff:ff:ff/128, src: n/a, gw: 2001:1234:5:9fff:ff:ff:ff:ff, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: Forgetting route: dst: 2001:1234:5:8f63::1/128, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main, proto: kernel, type: unicast
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: Forgetting route: dst: 2001:1234:5:8fff:ff:ff:ff:ff/128, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: Forgetting route: dst: 2001:1234:5:9fff:ff:ff:ff:ff/128, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: Forgetting route: dst: fe80::/64, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main, proto: kernel, type: unicast
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: Forgetting route: dst: n/a, src: n/a, gw: 2001:1234:5:8fff:ff:ff:ff:ff, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: Forgetting route: dst: 2001:1234:5:8f63::1/128, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: local, proto: kernel, type: local
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: Forgetting route: dst: fe80::3c56:ff:fe81:5740/128, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: local, proto: kernel, type: local
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: Forgetting route: dst: ff00::/8, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: local, proto: boot, type: unicast
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: Forgetting address: 2001:1234:5:8f63::1/128 (valid forever)
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: link_check_ready(): link is in pending state.
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: Kernel removed an address we don't remember: fe80::3c56:ff:fe81:5740/64 (valid forever), ignoring.
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: Kernel removed an address we don't remember: 149.10.124.58/28 (valid forever), ignoring.
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: Forgetting route: dst: 149.10.124.58/32, src: n/a, gw: n/a, prefsrc: 149.10.124.58, scope: host, table: local, proto: kernel, type: local
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: State changed: pending -> linger
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: Link removed
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: netdev removed
Oct 19 21:47:01 arch.localdomain systemd-networkd[168525]: dummy98: Created
```

**Comment by yuwata** [comment]
Ah,
```
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Addresses set
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 202.54.1.3/32, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main, proto: static, type: unreachable
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 149.10.123.3/32, src: n/a, gw: n/a, prefsrc: n/a, scope: link, table: local, proto: static, type: broadcast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 149.10.123.4/32, src: n/a, gw: n/a, prefsrc: n/a, scope: link, table: main, proto: static, type: multicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 202.54.1.2/32, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main, proto: static, type: blackhole
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 149.10.123.2/32, src: n/a, gw: n/a, prefsrc: n/a, scope: link, table: local, proto: static, type: anycast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 2001:1234:5:9fff:ff:ff:ff:ff/128, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 149.10.123.1/32, src: n/a, gw: n/a, prefsrc: n/a, scope: host, table: local, proto: static, type: local
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 192.168.1.2/32, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: n/a, src: n/a, gw: n/a, prefsrc: n/a, scope: link, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 202.54.1.4/32, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main, proto: static, type: prohibit
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 192.168.1.1/32, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 169.254.0.0/16, src: n/a, gw: n/a, prefsrc: n/a, scope: link, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 149.10.124.64/32, src: n/a, gw: n/a, prefsrc: n/a, scope: link, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 2001:1234:5:8fff:ff:ff:ff:ff/128, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: n/a, src: n/a, gw: 149.10.125.65, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: n/a, src: n/a, gw: 149.10.124.64, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 192.168.10.1/32, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 2001:1234:5:7fff:ff:ff:ff:ff/128, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: n/a, src: n/a, gw: 2001:1234:5:8fff:ff:ff:ff:ff, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Configuring route: dst: 192.168.10.2/32, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main, proto: static, type: unicast
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Could not add route: File exists
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Could not set routes: File exists
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: Failed
Oct 19 21:46:54 arch.localdomain systemd-networkd[168483]: dummy98: State changed: configuring -> failed
```

**Comment by yuwata** [comment]
(I really think `Restart=` should be disabled in the test suite. Otherwise, when `networkd` crashes, the logs of the next invocation may be shown. In the above case, pid 16843 is the problematic invocation, but pid 168483 is shown in the test output.)

**Comment by yuwata** [comment]
Hmm? `Restart=` is already disabled...
```
# /run/systemd/system/systemd-networkd.service.d/00-override.conf
[Unit]
StartLimitIntervalSec=0
[Service]
Restart=no
ExecStart=
ExecStart=!!/systemd-meson-build/systemd-networkd
Environment=SYSTEMD_LOG_LEVEL=debug
Environment=ASAN_OPTIONS="strict_string_checks=1:detect_stack_use_after_return=1:check_initialization_order=1:strict_init_order=1"
Environment=UBSAN_OPTIONS="print_stacktrace=1:print_summary=1:halt_on_error=1"
SystemCallFilter=
MemoryDenyWriteExecute=no
```

**Comment by yuwata** [comment]
Anyway, fix is waiting in #17400.
